File: factorialsProductTrailingZeros.c
int factorialsProductTrailingZeros(int l, int r) {
    int a=0,s=0;
    a = (l-1)/5 + (l-1)/25;
    for (;l<=r;l++) {
        //a=0;
        if (l%5==0) a++;
        if (l%25==0) a++;
        s+=a;
    }
    return s;
}

File: factorialsProductTrailingZeros.py
def factorialsProductTrailingZeros(l, r):
    x=1
    for i in range(l,r+1):
        x*=math.factorial(i)
    r=0
    d=list(str(x))
    d=d[::-1]
    for i in d:
        if i=='0':
            r+=1
        else:
            break
    return  r
